### PR TITLE
DOCS-1919-Add-port-requirement

### PR DESCRIPTION
**Pull request by bmckercher123**
Add port to egress gateway docs and main sys req doc. 
- Verify if this port is default or changeable. 
- Verify nodes where ports are required to be opened 

Product Version(s):
CE and CC all unarchived

Issue:
https://tigera.atlassian.net/browse/DOCS-1919

Link to docs preview:
https://deploy-preview-1056--tigera.netlify.app/calico-enterprise/next/getting-started/install-on-clusters/requirements
https://deploy-preview-1056--tigera.netlify.app/calico-enterprise/next/networking/egress/egress-gateway-on-prem
https://deploy-preview-1056--tigera.netlify.app/calico-enterprise/next/networking/egress/egress-gateway-azure
https://deploy-preview-1056--tigera.netlify.app/calico-enterprise/next/networking/egress/egress-gateway-aws

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [ ] Build completed successfully
- [ ] Test have passed

### CODE REVIEW

**Comment by bmckercher123** [comment]
@ctauchen Open to other wording. Also, agree that we should go back to all unarchived and fix for CE and CC?

File: calico-enterprise/_includes/components/ReqsSys.js [hunk 1]
```diff
@@ -378,6 +378,13 @@ function NetworkRequirementsEnt(props) {
             <td>Additional port required for Manager UI</td>
             <td>TCP 9449</td>
           </tr>
+         <tr className='odd'>
+            <td>
+              <strong>Egress gateway</strong>
+            </td>
+            <td>{prodname} egress gateway </td>
+            <td>TCP 4790 </td>
```
Comment (ctauchen) [inline calico-enterprise/_includes/components/ReqsSys.js]: Should that be UDP?

File: calico-enterprise/networking/egress/egress-gateway-aws.mdx [hunk 2]
```diff
@@ -205,6 +205,7 @@ an instance (for example when scaling up the cluster).
 **Required**
 
 - Calico CNI
+- Open port 4790 on the host
```
Comment (ctauchen) [inline calico-enterprise/networking/egress/egress-gateway-aws.mdx]: Do we need to specify TCP/UDP?

File: calico-enterprise/_includes/components/ReqsSys.js (hunk 1 above)
Comment (bmckercher123) [reply calico-enterprise/_includes/components/ReqsSys.js]: Yes. Fixed.

File: calico-enterprise/networking/egress/egress-gateway-aws.mdx (hunk 2 above)
Comment (bmckercher123) [reply calico-enterprise/networking/egress/egress-gateway-aws.mdx]: Yes. Fixed.

File: calico-cloud/_includes/components/ReqsSys.js [hunk 3]
```diff
@@ -359,6 +359,13 @@ function NetworkRequirementsEnt(props) {
             <td>Additional port required for Manager UI</td>
             <td>TCP 9449</td>
           </tr>
+          <tr className='odd'>
+            <td>
+              <strong>Egress gateway</strong>
+            </td>
+            <td>{prodname} compliance</td>
```
Comment (ctauchen) [inline calico-cloud/_includes/components/ReqsSys.js]: I think this needs to be about egw. A few other instances below

Comment (ctauchen) [reply calico-cloud/_includes/components/ReqsSys.js]: Line 366

**Review by ctauchen — COMMENTED** [review]
Needs a small change to a line in reqsys table in a few places. Needs requirements updates to docs proper for CE 3.15-3.18.

Pending changes, LGTM

File: calico-cloud/_includes/components/ReqsSys.js (hunk 3 above)
Comment (bmckercher123) [reply calico-cloud/_includes/components/ReqsSys.js]: Fixed.

**Pull request merged**